packages/lb-components/src/components/pointCloudView/PointCloudBackView.tsx

```diff
@@ -137,8 +223,15 @@ const PointCloudSideView = ({ currentData, config, checkMode }: IA2MapStateProps
 
     backPointCloudPolygonOperation.singleOn(
       'updatePolygonByDrag',
-      ({ newPolygon, originPolygon }: any) => {
-        backViewUpdateBox?.(newPolygon, originPolygon)
+      (updateList: UpdatePolygonByDragList) => {
+        if (ptCtx.selectedIDs.length === 1 && updateList.length === 1) {
+          const { newPolygon, originPolygon } = updateList[0];
+
+          if (newPolygon && originPolygon) {
+            transferPolygonDataToBoxParams(newPolygon, originPolygon);
```

Comment:
What does the function do?

Reply:
transfer PolygonData To BoxParams and rendering new box